docs/interpreting-experiments/confidence-intervals.md

```diff
@@ -1,34 +1,266 @@
+
+\\[10pt]
+
+\mu_{data} =&~~ \frac{m_C}{m_T}-1
+
+\\[10pt]
+
+\sigma^2_{data} =&~~ \frac{1}{m_C^2}\left(\frac{s_T^2}{N_T} +\frac{m_T^2}{m_C^2}\frac{s^2_C}{N_C}\right)
+\end{align}
+$$
+
+That is, the estimated lift, given data for $X$ from the control and treatment groups, is normally distributed around a mean of $\frac{m_C}{m_T} - 1$. For the frequentist methods (fixed-sample and sequential), that's where we end, in terms of estimating the lift (see below for turning that estimate into confidence intervals). For Bayesian, however, there's one more step.
+
+:::info
+
+If you are using CUPED, then the estimate of the lift will be a bit more complicated; see the [CUPED docs](cuped.md) for more information.
+
+:::
+
+
+#### Bayesian analysis: updating our prior
+
+In a Bayesian framework, you start with a *prior distribution*, which describes what you believe before running the experiment. Then, you run the experiment and collect data, which you use to *update* your prior: in essence, you combine your pre-experiment beliefs about what the lift would be, with the evidence you've gotten from the experiment, into a *new* set of beliefs, called the *posterior* (because it comes *after* gathering data). The estimated average lift is then just the mean of this posterior distribution.
+
+In our implementation of the Bayesian approach, we start the following prior on the lift:
+
+$$
+\begin{equation}
+P(\mathcal{L}) \sim \mathcal{N}(\mu_{pre}=0; ~\sigma_{pre}^2 = 0.1)
+\end{equation}
+$$
+
+In other words, our prior is that the lift, on average, will be zero, and that for each metric, about 50% of experiments will show a lift between -21% and +21%, and about 95% of experiments will show a lift between -62% and +62%; from our experience running experiments, this is a fairly conservative prior, as having lifts over ±50% is extremely rare.
+
+The evidence from the experiment is construed as a normal distribution $\hat{\mathcal{L}}$ just as with the frequentist methods (see equation 1 above). However, for the Bayesian method we use this evidence to update the above prior, and the result is our posterior.
+
+Specifically, our posterior is a normal distribution with mean $\mu_{post}$ and variance $\sigma_{post}^2$, where:
+
+$$
+\begin{align}
+\mu_{post} =&~~ \frac{
+  \frac{
+    \mu_{prior}
+  }{
+    \sigma^2_{prior}
+  } 
+  + 
+  \frac{
+    \mu_{data}
+  }{
+    \sigma^2_{data}
+  }
+}{
+  \frac{1}{\sigma^2_{prior}}
+  +
+  \frac{1}{\sigma^2_{data}}
+}
+
+\\[20pt]
+
+
+=&~~ \frac{1}{\sigma^2_{prior} + \sigma^2_{data}} \left(
+  \sigma^2_{prior}\mu_{data} + \sigma^2_{data}\mu_{prior}
+\right)
+
+&\text{\it\small{{(multiply top and bottom by $\sigma^2_{prior}\sigma^2_{data}$)}}}
+
+\\[10pt]
+
+\sigma^2_{post} =&~~ \frac{1}{\frac{1}{\sigma^2_{prior}} + \frac{1}{\sigma^2_{data}}}
+\end{align}
+$$
+
+Following [Gelman](http://www.stat.columbia.edu/~gelman/book/BDA3.pdf) (p50) (with a slight tweak), we can rewrite the posterior mean $\mu_{post}$ as:
```

Comment:
Unclear whether this is page of pdf or page of book -- might be better to refer to the equation. Also what is the slight tweak?